### PR TITLE
feat: introduce schema family and more

### PR DESCRIPTION
**Pull request by obdulia-losantos**
Related to https://jira.ehealthafrica.org/browse/AET-351

### CODE REVIEW

**Review by quesebifurcan — DISMISSED** [review]

**Comment by quesebifurcan** [comment]
Looks good!

File: aether-kernel/aether/kernel/api/views.py [hunk 1]
```diff
@@ -355,40 +389,119 @@ class SchemaViewSet(CustomViewSet):
...
         depth = request.query_params.get('depth')
         if depth:
             try:
                 depth = int(depth)
                 if depth > constants.LINKED_DATA_MAX_DEPTH:
-                    return Response({
-                        'description': f'Supported max depth is {constants.LINKED_DATA_MAX_DEPTH}'
-                    }, status=status.HTTP_400_BAD_REQUEST)
+                    return Response(
+                        {'description': f'Supported max depth is {constants.LINKED_DATA_MAX_DEPTH}'},
```
Comment (lordmallam) [inline aether-kernel/aether/kernel/api/views.py]: 🇩🇪 🇺🇸 🇳🇬

Comment (obdulia-losantos) [reply aether-kernel/aether/kernel/api/views.py]: :joy:

Comment (obdulia-losantos) [reply aether-kernel/aether/kernel/api/views.py]: @lordmallam fixed! :+1:

**Review by lordmallam — APPROVED** [review]

**Pull request merged**